source/diffHandler.py

```diff
@@ -70,31 +85,23 @@ def diff(self, newText: str, oldText: str) -> List[str]:
 				return []
 			with DiffMatchPatch._lock:
 				self._initialize()
-				old = oldText.encode("utf-8")
-				new = newText.encode("utf-8")
+				oldEncodedText = oldText.encode()

```

Comment:
Feel free to take or leave this suggestion, but I think `oldBytes` and `newBytes` might be more concise.